settings.gradle.kts

```diff
@@ -73,4 +100,8 @@ include("smallrye-openapi")
 
 include("spotless")
 
+if (false) {
```

Comment:
this looks like temporarily disabled, so should be commented imo
`// hack: used by dependabot but unused during build`